### PR TITLE
Description of the work division for committers.

### PR DESCRIPTION
**Pull request by annetill**
Signed-off-by: Anne Tilloy <anne.tilloy@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

This PR defines the work division between committers. This work division is going to evolve.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
 - Anne Tilloy (https://github.com/annetill)
 - Luis Zamarreno (https://github.com/zamarrenolm)
 
-Some committers are specialized in some field: please refer to [the maintainers table](MAINTAINERS.md) before submitting a pull request.
+#### Voting
+While the Powsybl organization aims to operate as a consensus based community, a TSC vote can be required to move the Organization forward. The voting members of the TSC will vote on a one vote per voting member basis. The simple majority is needed to approve proposals. A good way to vote is to create a poll [here](https://lists.lfenergy.org/g/powsybl-tsc/addpoll).
+
+#### Responsabilities
```
Comment (agnesLeroy) [inline CONTRIBUTING.md]: Responsibilites

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
...
+The Powsybl organization contains several repositories. We have at least a committer in charge by repository. By "in charge", we mean:
```
Comment (agnesLeroy) [inline CONTRIBUTING.md]: The Powsybl project comprehends several repositories. There is at least one committer in charge of each repository. By "in charge", we mean:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
...
+-	Best effort to review the pull request,
+-	Best effort to resolve issues,
+-	Building and publishing the releases, including writing the release notes and informing the community,
+-	In case of impossibility, the committer in charge has to ask the TSC through the list [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org) to find another committer to review the pull request, resolve the issue or build and publish the release.  
+
+Please refer to our [maintainers file](MAINTAINERS.md) to have more details about our work division.
```
Comment (agnesLeroy) [inline CONTRIBUTING.md]: for more details

File: MAINTAINERS.md [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+
+**powsybl.github.io**
+- Review: all committers
+
+**powsybl-core**
+- Review: [mathbagu](https://github.com/mathbagu), [MioRtia](https://github.com/MioRtia), [zamarrenolm](https://github.com/zamarrenolm) and [annetill](https://github.com/annetill)
+- Release: [mathbagu](https://github.com/mathbagu) or [MioRtia](https://github.com/MioRtia)
+
+**powsybl-open-loadflow**
+- Review: [geofjamg](https://github.com/geofjamg)
```
Comment (mathbagu) [inline MAINTAINERS.md]: Add Geoffroy and I as releaser

File: MAINTAINERS.md [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+
+**powsybl-dynawo**
+- Review: [mathbagu](https://github.com/mathbagu)
```
Comment (mathbagu) [inline MAINTAINERS.md]: Add Myself as a releaser

File: MAINTAINERS.md [hunk 6]
```diff
@@ -1,14 +1,52 @@
...
+(coming soon)
+-	Review: [pl-buiquang](https://github.com/pl-buiquang) and [geofjamg](https://github.com/geofjamg)
+-	Release: [pl-buiquang](https://github.com/pl-buiquang) (release notes) and [mathbagu](https://github.com/mathbagu) or [MioRtia](https://github.com/MioRtia) (release upload)
+
+**powsybl-hpc**
+-	Review: [sylvlecl](https://github.com/sylvlecl) (Slurm) and [geofjamg](https://github.com/geofjamg) (MPI)
+-	Release: [sylvlecl](https://github.com/sylvlecl)
+
+**powsybl-gse**
+-	Review:[pl-buiquang](https://github.com/pl-buiquang) and [geofjamg](https://github.com/geofjamg)
```
Comment (mathbagu) [inline MAINTAINERS.md]: To be verified: does Github render well lists with spaces between the bullet and the text. Do be consistent, do we remove the extra spaces for the 3 following items?!

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
 - Anne Tilloy (https://github.com/annetill)
 - Luis Zamarreno (https://github.com/zamarrenolm)
 
-Some committers are specialized in some field: please refer to [the maintainers table](MAINTAINERS.md) before submitting a pull request.
+#### Voting
+While the Powsybl organization aims to operate as a consensus based community, a TSC vote can be required to move the Organization forward. The voting members of the TSC will vote on a one vote per voting member basis. The simple majority is needed to approve proposals. A good way to vote is to create a poll [here](https://lists.lfenergy.org/g/powsybl-tsc/addpoll).
+
+#### Responsibilities
+The Powsybl project comprehends several repositories. There is at least one committer in charge of each repository. By "in charge", we mean:
```
Comment (mathbagu) [inline CONTRIBUTING.md]: Are you sure of this term « comprehends »? I would use « split into »

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
...
+-	Best effort to review the pull request,
+-	Best effort to resolve issues,
+-	Building and publishing the releases, including writing the release notes and informing the community,
+-	In case of impossibility, the committer in charge has to ask the TSC through the list [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org) to find another committer to review the pull request, resolve the issue or build and publish the release.
```
Comment (mathbagu) [inline CONTRIBUTING.md]: See below about spaces between the bullet and the text

**Comment by mathbagu** [comment]
You forget to mention powsybl-math-native. You can put Geoffroy and I as reviewer/releaser ;)

**Pull request merged**